web/flux_to_redux/simple/examples/influx_implementation/components/connect_flux_big_block.dart

```diff
@@ -0,0 +1,40 @@
+import 'package:over_react/over_react.dart';
+import 'package:over_react/over_react_flux.dart';
+import 'package:over_react/over_react_redux.dart';
+
+import '../store.dart';
+
+part 'connect_flux_big_block.over_react.g.dart';
+
+UiFactory<ConnectFluxBigBlockProps> ConnectedConnectFluxBigBlock =
+    connectFlux<FluxStore, RandomColorActions, ConnectFluxBigBlockProps>(
+  mapStateToProps: (state) => (ConnectFluxBigBlock()..backgroundColor = state?.backgroundColor),
+  mapActionsToProps: (actions) =>
+      (ConnectFluxBigBlock()..changeBackgroundColor = () => actions.changeBackgroundColor()),
```

Comment:
```suggestion
      (ConnectFluxBigBlock()..changeBackgroundColor = actions.changeBackgroundColor),
```

when you do it as an inline function it will be a "new" instance every time and thus trigger a re-render even if props are identical.

Reply:
With Inline Function:
![Screen Shot 2020-01-02 at 2 32 54 PM](https://user-images.githubusercontent.com/41018730/71694586-d9156f80-2d6c-11ea-9295-7c5f5301bb1e.png)

Without inline function:
![Screen Shot 2020-01-02 at 2 32 18 PM](https://user-images.githubusercontent.com/41018730/71694592-dca8f680-2d6c-11ea-835b-a0ff8c8fd8ea.png)